web/app/scripts/src/libraries.es.js

```diff
@@ -0,0 +1 @@
+export const angular = window.angular;
```

Comment:
Hmm yes. Thats sounds like it could eventually be a very nice way to do it if we browserify everything and link with npm packages, but you are right it would be a big change from where we are now